migrations/create_brick_devices_table.php.stub

```diff
@@ -12,8 +12,7 @@ class CreateBrickDevicesTable extends Migration
     {
         Schema::create('brick_devices', function (Blueprint $table) {
             $table->increments('id');
```

Comment:
Perhaps good to change this to a `$table->id();` as well. 

```suggestion
            $table->id();
```

Reply:
Updated that too. Good catch. Thanks!